packages/amplify-provider-awscloudformation/lib/initialize-env.js

```diff
@@ -47,8 +47,7 @@ function run(context, providerMetadata) {
         });
       });
 
-      const jsonString = JSON.stringify(amplifyMeta, null, 4);
-      fs.writeFileSync(amplifyMetafilePath, jsonString, 'utf8');
+      context.amplify.writeObjectAsJson(amplifyMetafilePath, amplifyMeta, true);
```

Comment:
if it's not too much trouble, would be nice to use JSONUtilities here and in the placed you modified in push-resources. But we could also come back and refactor these files later.

Reply:
I'm going to leave as is for now. Each file deserves a separate thorough refactor to use `JSONUtilities` and `StateManager` from cli core.